### ISSUE TITLE
Line 44 breaks on CircleCI

### ISSUE THREAD

**Issue by maxwofford** [hackclub/markcop#5]
CircleCI tests fail when running markcop:

![screenshot from 2015-09-28 16 27 52](https://cloud.githubusercontent.com/assets/5891442/10151588/e519b9d4-65fd-11e5-9db0-ddaf5ef568b6.png)

I've replicated the problem locally by using dash (instead of bash):

```
../markcop/bin/markcop: 44: ../markcop/bin/markcop: Syntax error: word unexpected (expecting ")")
```

This occurs  on line 52 also.

EDIT: Somehow #3  fixes this: https://circleci.com/gh/hackedu/hackedu/337

![screenshot from 2015-09-28 16 33 02](https://cloud.githubusercontent.com/assets/5891442/10151650/9ef0ba06-65fe-11e5-92d1-39192ec75282.png)

**Comment by maxwofford** [comment]
This is closed by #3

**Comment by zachlatta** [comment]
Closed by #3.
